pkg/k8shandler/deployment.go

```diff
@@ -242,13 +237,15 @@ func (node *deploymentNode) setPaused(paused bool) error {
 	// we use pauseNode so that we don't revert any new changes that should be made and
 	// noticed in state()
 	pauseNode := node.self.DeepCopy()
+	ll := log.WithValues("node", pauseNode.Name)
 
 	nretries := -1
 	retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 		nretries++
-		if getErr := node.client.Get(context.TODO(), types.NamespacedName{Name: pauseNode.Name, Namespace: pauseNode.Namespace}, pauseNode); getErr != nil {
-			logrus.Debugf("Could not get Elasticsearch node resource %v: %v", pauseNode.Name, getErr)
-			return getErr
+		if err := node.client.Get(context.TODO(), types.NamespacedName{Name: pauseNode.Name, Namespace: pauseNode.Namespace}, pauseNode); err != nil {
+			ll.Info("Could not get Elasticsearch node resource",
```

Comment:
why `Info` over `Error` here?

Reply:
Because this is in a retry loop. It's "handled" by the retry loop so it's just informational.